retriever/lib/install.py

```diff
@@ -1,209 +0,0 @@
-from __future__ import print_function
-from __future__ import absolute_import
-import imp
-import os
-import sys
-from pkg_resources import parse_version
-from retriever.engines import choose_engine
-from retriever.lib.compile import compile_json
-from retriever.lib.defaults import DATA_DIR, SCRIPT_SEARCH_PATHS, VERSION
-from retriever.lib.tools import name_matches
-from retriever.lib.scripts import SCRIPT_LIST
-
-script_list = SCRIPT_LIST()
```

Comment:
I think check_for_updates() is supposed to be run by the code. This is true if you use `python setup.py install`. however, it is not true for `pip install . -U`. So if we can change and make sure that if `install` is to run, we need to update or check or check if there are scripts.